src/pyocr/tesseract.py

```diff
@@ -331,7 +317,7 @@ def image_to_string(image, lang=None, builder=None):
 
     Arguments:
         image --- image to OCR
-        lang --- tesseract language to use
+        lang --- tesseract language to use. 
```

Comment:
PEP8:White space at the end of the line
+ you added a point, but there is none on the other lines